pkg/controller/infrastructure/actuator.go

```diff
@@ -61,3 +64,29 @@ func generateTerraformInfraVariablesEnvironment(credentials *aws.Credentials) ma
 		"TF_VAR_SECRET_ACCESS_KEY": string(credentials.SecretAccessKey),
 	}
 }
+
+// NewDefaultInitializer retuns a initializer for the configuration and variables whiout modifying the state
+func NewDefaultInitializer(c client.Client, release *chartrenderer.RenderedChart, infrastructure *extensionsv1alpha1.Infrastructure) (terraformer.Initializer, error) {
```

Comment:
Does it make sense to put these two functions into the `gardener/gardener/extensions/pkg/terraformer` package so that it is not needed to duplicate it into every extensions repo?

Reply:
This function does not exists anymore